components/kyma-environment-broker/internal/eu_access/whitelisted_globalaccounts_test.go

```diff
@@ -0,0 +1,18 @@
+package eu_access
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReadWhitelistedGlobalAccountIdsFromFile(t *testing.T) {
+	// given/when
+	d, err := ReadWhitelistedGlobalAccountIdsFromFile("test/eu_access_whitelist.yaml")
```

Comment:
Can we add case where we test multiple entries of whitelisted accounts?

Reply:
Done.